File: DTOs/UpdateDTO/UserUpdateDto.cs
namespace BackendBookstore.DTOs.UpdateDTO
{
    public class UserUpdateDto
    {
        public int UsersId { get; set; }

        public string? UserRole { get; set; }

        public string Username { get; set; } = null!;

        public string PasswordLogin { get; set; } = null!;

        public string FirstName { get; set; } = null!;

        public string LastName { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string? Phone { get; set; }

        public char? Genre { get; set; }
    }
}

File: FrontendBookstore/src/dto/user/UserUpdateDto.ts
import UserRole from "../../model/UserRole";

export interface UserUpdateDto {
  usersId: number;
  userRole: UserRole; // or an enum if you have defined one
  username: string;
  firstName: string;
  lastName: string;
  email: string;
  phone?: string;
  genre?: string; // or char if you have defined one
}
